### PR TITLE
Improvements to UG

### PR DESCRIPTION
**Pull request by twothicc**
Addresses #319 

- Added filter_interview anchor link to table of contents
- Added diagram to show parts of an applicant card
- Add sample workflow to quickstart with illustrations
- Change color of markdown block to be different for add and find input specifications

### CODE REVIEW

**Review by kaushikkrdy — CHANGES_REQUESTED** [review]
Looks good except for a few minor things!

* I like the new description of walking the user through the scenario. However, do you think it could be considered too wordy for a 'Quick Start'?

* If the user were to follow all the commands in order in Quick Start, they would run into an error once they reach step 6. This is because the user might be likely to miss the 'Suppose time passes and you now have more applicants in your list` sentence in 5.6 and they might not add any of their own as they are simply looking at the code portions.
    * `delete 2` will return an error since only `Mary` is in the list
    * `unmark 1` will return an error since only `Mary` is in the list and `Mary` is unmarked
    * `unmark 1` description of unmarking `John` would be inaccurate

Would it be possible to make it clearer so that the user does not run into an error?

**Review by kaushikkrdy — APPROVED** [review]
LGTM! The user is less likely to be confused with the updated description of the workflow.

**Comment by twothicc** [comment]
Thanks for the review!

I think that the sample workflow only appears long because of the large size of pictures. However, I feel that it is necessary to show the user the full state of the application after major checkpoints in the workflow to ensure that the user does not get lost.

The example command descriptions are not updated from last time. Good call! I fixed it and added a warning informing users that these example commands are not related to the sample workflow.
